src/components/MarkdownMusicSourceFetcher.test.js

```diff
@@ -3,7 +3,7 @@ import ReactDOM from 'react-dom';
 import MarkdownMusicSourceFetcher from './MarkdownMusicSourceFetcher';
 import { BrowserRouter as Router, Route } from 'react-router-dom';
 
-it('renders without crashing', () => {
+it('renders MarkdownMusicSourceFetcher without crashing', () => {
```

Comment:
Should these instead be part of `describe()`? Would help with any future tests we add.

```javascript
describe('MarkdownMusicSourceFetcher', () => {
  it('renders...', () => {});
};
```